folium/plugins/polyline_offset.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from branca.element import JavascriptLink
+
+from folium.vector_layers import path_options
+from folium.vector_layers import PolyLine
+
+
+class PolyLineOffset(PolyLine):
+    """
+    Class adding offset capabilities to the PolyLine class.
+
+    This plugin adds to folium Polylines the ability to be drawn with a
+    relative pixel offset, without modifying their actual coordinates. The offset
+    value can be either negative or positive, for left- or right-side offset,
+    and remains constant across zoom levels.
+
+    See :func:`folium.vector_layers.path_options` for the `Path` options.
+
+    Parameters
+    ----------
+    locations: list of points (latitude, longitude)
+        Latitude and Longitude of line (Northing, Easting)
+    popup: str or folium.Popup, default None
+        Input text or visualization for object displayed when clicking.
+    tooltip: str or folium.Tooltip, optional
+        Display a text when hovering over the object.
+    offset: int, default 0
+        Relative pixel offset to draw a line parallel to an existant one,
+        at a fixed distance.
+    **kwargs:
+        Polyline options. See their Github page for the
+        available parameters.
+
+    See https://github.com/bbecquet/Leaflet.PolylineOffset
+
+    Examples
```

Comment:
Good that you include example. They can be shorter though I think, no need to include the `folium.Map` stuff.

Maybe:
`>>> PolyLineOffset([[58, -28], [53, -23]], color='#f00', offset=-5).add_to(m)`

(In general we don't use `map` as variable name, because that interferes with Pythons `map` operator. Convention is to use `m` instead.)

Reply:
- Renamed the `map `into `m`
- Reduce the number of lines of the example